pilot/pkg/xds/debug.go

```diff
@@ -378,11 +378,13 @@ func (s *DiscoveryServer) endpointz(w http.ResponseWriter, req *http.Request) {
 	writeJSON(w, resp, req)
 }
 
+var DistributionTrackingDisabledMessage = "Pilot Version tracking is disabled.  Please set the " +
+	"PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING environment variable to true to enable."
+
```

Comment:
I suggest that change the message as below:
```
"Pilot Version tracking is disabled.  Please enable it by setting " +
	"PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING environment variable as true."
```

Reply:
I don't really see any difference between the messages. @ericvn Could you give any suggestions? Thanks.

Reply:
The message is basically the same. But I do prefer the `... disabled. To enable ...`.

And I would say `setting to` true instead of `setting as`.

So I might make it something like:
```
"Pilot Version tracking is disabled. It may be enabled by setting the " +
	"PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING environment variable to true."
```